python_modules/dagster/dagster/core/runs.py

```diff
@@ -0,0 +1,90 @@
+from collections import namedtuple, OrderedDict
+import json
+import os
+import shutil
+
+from dagster import check
+from dagster.utils import mkdir_p, list_pull
+
+
+def base_run_directory():
+    # TODO what is going to be the best place for this on windows?
```

Comment:
something like

```
if sys.platform == 'win32':
    return os.path.join(tempfile.gettempdir(), 'dagster', 'runs')
return '/tmp/dagster/runs'
```

Reply:
are you sure? doesn't gettempdir() just get some random machine generated thing? we want a location that is stable between invocations

Reply:
on windows it does what we want, on os x it gives a bad value because $TMPDIR is set per-process (and therefore not stable).